images/helm-controller/tests/02-flux-install.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# monopod:tag:k8s
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+if [[ "${IMAGE_NAME}" == "" ]]; then
+    echo "Must set IMAGE_NAME environment variable. Exiting."
+    exit 1
+fi
+
+IMAGE_REPOSITORY=$(echo "${IMAGE_NAME}" | cut -d: -f1)
+IMAGE_TAG="$(echo "${IMAGE_NAME}" | cut -d: -f2)"
```

Comment:
the cut is unreliable since there is a port in the image name (3 ":" characters)

these vars should be handed to you in the env for k8s tests, so you dont need them
```suggestion
```

```
+ export IMAGE_REGISTRY=registry.local:5000
+ IMAGE_REGISTRY=registry.local:5000
+ export IMAGE_REPOSITORY=helm-controller
+ IMAGE_REPOSITORY=helm-controller
+ export IMAGE_TAG=latest
+ IMAGE_TAG=latest
+ export IMAGE_NAME=registry.local:5000/helm-controller:latest
+ IMAGE_NAME=registry.local:5000/helm-controller:latest
+ export IMAGE_TAG_SUFFIX=
+ IMAGE_TAG_SUFFIX=
```

Reply:
ack, bad copy/pasta 😓